package.json

```diff
@@ -45,6 +45,7 @@
     "body-parser": "^1.12.0",
     "buffer-equal-constant-time": "^1.0.1",
     "chai-events": "0.0.1",
+    "coffee-coverage": "^3.0.0",
```

Comment:
We no longer need coffee-coverage actually, because the tests build to javascript and the coverage is run over those files with the maps.